docs/api/IoT.js-API-Assert.md

```diff
@@ -5,203 +5,234 @@ The following shows Assert module APIs available for each platform.
-        - [Properties](#properties)
-            - [`actual`](#actual)
-            - [`message`](#message)
-            - [`expected`](#expected)
-            - [`name`](#name)
-            - [`operator`](#operator)
+You can access the functions of the module by adding `require('assert')` to your file.
 
 
-# Assert
+## Class: AssertionError
+
+Assert module will produce `AssertionError` in case of an assertion failure. `AssertionError` inherits standard `Error` thus it has properties provided by `Error` object including additional properties.
+
+
+### Properties
+
+
+* `actual` {any} This property contains the actual value.
 
-Assert module is for writing test for applications.
+* `expected` {any} This property contains the expected value.
 
-You can get assert module by `require('assert')` the module provide.
+* `message` {any} The error message, default value is the error itself.
+
+* `name` {String} The name is set AssertionError.
+
+* `operator` {String} This property contains the operator used for evaluating `actual` with `expected`.
 
 
 ## Module Functions
 
 
-### `assert.assert(value[, message])`
-* `value <any>`
-* `message <any>`
+### assert(value[, message])
+* `value` {any} Value to test.
+* `message` {any} Message displayed in the thrown error.
 
-Test if `value` is true, it throws an `AssertionError` exception if `value` evaluates `false`.
+Checks if the `value` is truthy. If not, throws an AssertionError, with the given `message`.
 
 **Example**
 
 ```js
 var assert = require('assert');
-assert.assert(1 == 1);
+
+assert.assert(1);
+// OK
+
+assert.assert(true);
+// OK
+
+assert.assert(false);
+// throws "AssertionError: false == true"
+
+assert.assert(0);
+// throws "AssertionError: 0 == true"
+
+assert.assert(false, "it's false");
+// throws "AssertionError: it's false"
 ```
 
 
-### `assert.doesNotThrows(block[, message])`
-* `value <Function()>`
-* `message <any>`
+### doesNotThrow(block[, message])
+* `block` {Function} Expects a function.
+* `message` {any} Message to be displayed.
 
-Test if the given `block` does not throw any exception, if not, it throws an exception.
+Tests if the given `block` does not throw any exception, if it does, throws an
+exception with the given `message`.
 
 **Example**
 
 ```js
 var assert = require('assert');
+
 assert.doesNotThrow(
   function() {
-    assert.assert(1 == 1);
+    assert.assert(1);
   }
 );
+// OK
+
+assert.doesNotThrow(
+  function() {
+    assert.assert(0);
+  }
+)
+// throws "AssertionError: Got unwanted exception."
 ```
 
 
-### `assert.equal(actual, expected[, message])`
-* `actual <any>`
-* `expected <any>`
-* `message <any>`
+### equal(actual, expected[, message])
+* `actual` {any} The actual value.
+* `expected` {any} The expected value.
+* `message` {any} Message to be displayed.
 
-Test if `actual == expected` is evaluated to `true`, if not, it throws an exception.
+Tests if `actual == expected` is evaluated to `true`, if not, throws an
```

Comment:
evaluated to `true`. If it does not, the function throws
given optional `message`.